recipes/cfm/meta.yaml

```diff
@@ -1,26 +1,25 @@
+{% set name = "cfm" %}
+{% set version = "2.4.1" %}
+
 package:
-  name: cfm
-  version: "33"
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
 
 source:
-  # the download is a mirror of a snapshot of 
-  # https://sourceforge.net/code-snapshots/svn/c/cf/cfm-id/code/cfm-id-code-r33-cfm.zip
-  # which is only present for ~24h
-  #
   # patch: - comment tests with missing data
   # - bugfix in cfm-code/EM.cpp (cerr -> endl)
   # - fix for tools that try to access ISOTOPE.DAT or the tests folder at the working 
   #   dir (prepend $CFM_OUTPUT_DIR with the help of macros)
-  - url: https://depot.galaxyproject.org/software/cfm/cfm_r33_src_all.zip
-    sha256: 6518e70d2acf995d95d610312b234b5a69ee13ba0eed4836235ce32d73e32e35
+  - url: https://bitbucket.org/wishartlab/cfm-id-code/get/CFM-ID_{{ version }}.tar.gz
+    sha256: 0ddce9b847063459c8c9ca6fc96fcabec29aced6e936b47c2e59e84bb3b3b6f1
     patches:
       - source.patch
```

Comment:
Seems that this patch does not work anymore?

Reply:
Different line endings. Hunk #1 FAILED at xx (different line endings).